test/src/native_updater_test.dart

```diff
@@ -0,0 +1,157 @@
+import 'package:flutter/cupertino.dart';
```

Comment:
can you please split test files into 1 for iOS and 1 for Android at least?

Reply:
Done.

Reply:
Resolved on 876d9ef